src/d1to2fix/main.d

```diff
@@ -102,6 +110,17 @@ int main ( string[] args )
         return 1;
     }
 
+    // disable all warnings from dsymbol
+
+    import std.experimental.logger;
+    globalLogLevel = LogLevel.error;
+
+    // Build symbol cache for parsed modules
+
+    import d1to2fix.symbolsearch;
+    config.importPaths ~= "/usr/include/d2/dmd-transitional/";
```

Comment:
I can't imagine any other place where to put it unless we want `d1to2fix` to parse `dmd.conf` - makd can't provide it either.